server/src/main/java/com/cloud/network/element/ConfigDriveNetworkElement.java

```diff
@@ -478,7 +476,7 @@ private boolean deleteConfigDriveIso(final VirtualMachine vm) throws ResourceUna
         return true;
     }
 
-    private void addConfigDriveDisk(final VirtualMachineProfile profile, final DataStore dataStore) throws ResourceUnavailableException {
+    public static void addConfigDriveDisk(final VirtualMachineProfile profile, final DataStore dataStore) throws ResourceUnavailableException {
```

Comment:
can you explain? I don't see it being used anywhere

Reply:
@Spaceman1984 have you seen this question?

Reply:
Must be a leftover artifact from long days of experimentation, will remove.